linux/check_os_fullname/miracle_fullname_map.yml

```diff
@@ -3,15 +3,15 @@
 ---
 # Guest id "asianux8_64Guest" and full name "MIRACLE LINUX 8 (64-bit)" is available on ESXi 6.7P06 or later
 
-- name: "Set fact of expected guest id on ESXi {{ esxi_version }}"
-  ansible.builtin.set_fact:
-    expected_guest_major_ver: "{{ [guest_os_ansible_distribution_major_ver | int, 8 ] | min }}"
-
 - name: "Set facts when vmtools_version is larger than 12.3.5"
   when:
     - vmtools_version is defined
     - vmtools_version is version('12.3.5', '>')
   block:
+    - name: "Set fact of expected guest id on ESXi {{ esxi_version }}"
```

Comment:
The task name is not proper, please change to "Set fact of expected guest OS major version on ESXi {{ esxi_version }}"

Reply:
I saw Miracle Linux 9.x is under testing. Maybe you can wait it's finished, and then add Miracle Linux 9.x into README if there is no blocker issue.

Reply:
Running testcase check_os_fullname with this branch for Miracle linux 9.2 ....